evaluators/rank/NdcgEvaluator/__init__.py

```diff
@@ -0,0 +1,52 @@
+from math import log
+from typing import Sequence, Any
+
+from jina.executors.evaluators.rank import BaseRankingEvaluator
+
+class NDCGEvaluator(BaseRankingEvaluator):
+    """
+    :class:`NDCGEvaluator` evaluates normalized discounted cumulative gain for information retrieval.
+    """
+
+    @property
+    def metric(self):
+        return f'nDCG@{self.eval_at}'
+
+    def evaluate(
+            self,
+            actual: Sequence[Any],
+            desired: Sequence[Any],
+            use_traditional_formula=False,
```

Comment:
I do not like this name, seems like I need to know history to use this executor.

Reply:
any suggestions?

Reply:
Power_relevance?

Reply:
I agree.. `use_traditional_formula ` sounds wired.. since we don't really have a name for these two types of formulars.. some poeple use `e` as log base, some people use `2`. A lot of things were not standardised. @JoanFM